src/vaults/BalanceSheet.sol

```diff
@@ -141,6 +146,7 @@ contract BalanceSheet is Auth, Recoverable, IBalanceSheet, IBalanceSheetGatewayH
         external
         authOrManager(poolId)
     {
+        require(!root.endorsed(from), CannotTransferFromEndorsedContract());
```

Comment:
Blocks balance sheet managers from transferring shares out of the global escrow

Reply:
1 bug fix for this, to block revoking shares in the global escrow: https://github.com/centrifuge/protocol-v3/pull/321/commits/5ce3316e7de9ff008841700f49790227e1ca3f3d

Reply:
Great catch!